packages/paste-website/src/pages/components/form-pill-group/index.mdx

```diff
@@ -0,0 +1,510 @@
+      webkitallowfullscreen
+      mozallowfullscreen
+      allowfullscreen
+      title="Silent demo on keyboard navigation"
+    ></iframe>
+  </AspectRatio>
+</Box>
+
+#### Display Pill vs. Form Pill
+
+There are some very important semantic differences between a Display Pill and Form Pill Group. Below is a table explaining the main differences to get a better understanding of when to use each type of pill.
+
+<FormPillVsDisplayPillTable />
+
+## Examples
+
+### Basic Form Pill Group
+
+The basic Form Pill Group can be composed by adding `FormPill` components as children of `FormPillGroup`. These must be used with the `useFormPillState` hook, and the returned state should be spread onto each component.
+
+This will provide you with some internal state management and keyboard navigation.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {defaultExample}
+</LivePreview>
+
+### Selectable Form Pill Group
+
+To make pills inside the Pill Group selectable, manage the selection state yourself and combine it with the state returned from the `useFormPillState` hook.
+
+To do so, track which pill is selected in a separate store of state. When rendering the Pill Group, cross reference the rendered pills with the selection state, and conditoinally set `selected` on each `FormPill` that requires it.
+
+The `onSelect` event handler will fire when ever the pill is clicked, or the spacebar or enter key is pressed. Use this to respond to your users' selection interactions.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {selectableExample}
+</LivePreview>
+
+### Removable Form Pill Group
+
+Form Pills are given a closing "x" button when provided an `onDismiss` event handler. Because the Form Pill Group is largely presentational, you can provide "removable" functionality by managing the state of the rendered pills. By responding to user interactions and hooking into the `onDismiss` event handler, the rendered state of the Form Pill Group can be updated and pills can be selectively removed from the collection.
+
+The `onDismiss` event handler will fire when a user clicks on the close x, or presses their backspace or delete key when foused on a pill.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon, Text}} noInline>
+  {removableExample}
+</LivePreview>
+
+### Simple Multiselect Combobox Example
+
+Using Form Pill Group, along with the [Combobox Primitive](/primitives/combobox-primitive#multiselect-combobox-example) you can easily compose an interactive multiple select form field.
+
+<LivePreview
+  scope={{
+    useComboboxPrimitive,
+    useMultiSelectPrimitive,
+    Box,
+    Label,
+    Input,
+    useUID,
+    useUIDSeed,
+    ComboboxListbox,
+    ComboboxListboxGroup,
+    ComboboxListboxOption,
+    FormPill,
+    FormPillGroup,
+    useFormPillState,
+  }}
+  noInline
+  language="jsx"
+>
+  {multiSelectExample}
+</LivePreview>
```

Comment:
Any chance you could make this autocomplete, or put the chevron in the field?

Otherwise it looks like customer should be able to type in the field, but it doesn't allow for it.

Reply:
Hmmm this is turning out to be way more scope than just Pill docs to get this to work. So two choices:

1) example stays as is
2) example is removed.